common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -166,7 +166,7 @@ def get_course_date(self):
         """
         Get course date of the first course from dashboard
         """
-        return self.q(css='ul.listing-courses .course-item:first-of-type .info-date-block').first.text[0]
+        return self.q(css='ul.listing-courses .course-item:first-of-type .info-date-block localized-datetime').text
```

Comment:
I think this is going to have to change. There was an issue with this method in my bok choy tests a commit ago, but I'll get it working.